pkg/config/config.go

```diff
@@ -158,6 +158,13 @@ type ContainersConfig struct {
 	// PidNS indicates how to create a pid namespace for the container
 	PidNS string `toml:"pidns,omitempty"`
 
+	// Copy up the content from underlying image into the newly created
+	// volume when the container is created as opposed to when it is
+	// started. If false, the container engine will not copy up the
+	// content until the container is started. Setting it to true may
+	// have negative performance implications.
```

Comment:
```
// Copy the content from the underlying image into the newly created 
// volume when the container is created instead of when it is started.
// If `false`, the container engine will not copy the content until the
// container is started. Setting it to `true` may have negative
// performance implications.
```